src/components/Provider/ProviderHeader/CopyTraderButton/CopyTraderButton.js

```diff
@@ -48,9 +68,31 @@ const CopyTraderButton = ({ provider }) => {
     showStopCopyingModal(false);
   };
 
-  const followingFrom = exchangeConnections.find(
-    (e) => e.internalId === provider.exchangeInternalId,
-  );
+  const cancelDisconnect = () => {
+    showCancelDisconnectLoader(true);
+    const payload = {
+      token: storeSession.tradeApi.accessToken,
+      providerId: provider.id,
+      internalExchangeId: selectedExchange.internalId,
+    };
+    tradeApi
+      .providerCancelDisconnect(payload)
+      .then(() => {
+        const providerPayload = {
+          token: storeSession.tradeApi.accessToken,
+          providerId: provider.id,
+          version: 2,
+        };
+        dispatch(setProvider(providerPayload));
+        dispatch(showSuccessAlert("", "Your disconnection has been cancelled!"));
```

Comment:
We need to translate that

Reply:
@cwagner22  I was thinking about this one, I completely missed it.